Argo/Functions/catDecoded.swift

```diff
@@ -12,6 +26,20 @@ public func catDecoded<T>(xs: [Decoded<T>]) -> [T] {
   return accum
 }
 
+/**
+  Remove `Failure` values from a Dictionary and unwrap `Success`es.
```

Comment:
also these ... would it be more "correct" to say "Filter out" or something like that?